src/lib/datepicker/datepicker.ts

```diff
@@ -142,7 +142,14 @@ export class MatDatepicker<D> implements OnDestroy {
    * Whether the calendar UI is in touch mode. In touch mode the calendar opens in a dialog rather
    * than a popup and elements have more padding to allow for bigger touch targets.
    */
-  @Input() touchUi = false;
+  @Input()
+  get touchUi() {
+    return this._touchUi;
+  }
+  set touchUi(value: any) {
```

Comment:
I think its best to leave the types as `boolean` that way when people `@ViewChild` this and try to programmatically set it they at least know that they should be passing a boolean (and then behind the scenes we coerce it anyways it just in case it came from a template binding).

Reply:
I just followed the pattern that disabled had right below it. I'll update both.

Reply:
yeah I think there's a lot that use `any`, but we can use the expected type for new ones we add and maybe eventually fix all the `any` ones